### ISSUE TITLE
Character wrapping input line in linux

### ISSUE THREAD

**Issue by daephx** [JanDeDobbeleer/oh-my-posh#1035]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

Text is wrapping when it shouldn't inside of bash. It does not seem to be an issue in pwsh.
![ygoHjvjevZ](https://user-images.githubusercontent.com/36192863/136691075-c2dda1eb-f8d3-4266-82d0-15709c95959c.gif)

I have observed this problem inside Windows Terminal running wsl/ubuntu.
It also seems to occur on my ubuntu laptop under visual studio code.
An interesting thing is that my screen on the laptop is much smaller, so the area  of effect is much smaller.
e.i. If the file pane is open? It takes only about 20 `aaa`'s before the text starts to wrap 

### Version

bash: $ oh-my-posh --version `5.7.3`
release: posh-linux-amd64

### Theme

Modified `slim.omp.json`:
```jsonc
{
    "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
    "final_space": false,
    "console_title": true,
    "console_title_style": "template",
    "console_title_template": "{{if .Root}}root :: {{end}}{{.Shell}} :: {{.Folder}}",
    "blocks": [
        {
            "type": "prompt",
            "alignment": "left",
            "segments": [
                {
                    "type": "os",
                    "style": "diamond",
                    "leading_diamond": "\ue0c7\u2593",
                    "foreground": "#26C6DA",
                    "background": "#2f2f2f",
                    "properties": {
                        "alpine": "\uf300",
                        "arch": "\uf303",
                        "centos": "\uf304",
                        "debian": "\uf306",
                        "elementary": "\uf309",
                        "fedora": "\uf30a",
                        "gentoo": "\uf30d",
                        "linux": "\ue712",
                        "macos": "\ue711",
                        "manjaro": "\uf312",
                        "mint": "\uf30f",
                        "opensuse": "\uf314",
                        "raspbian": "\uf315",
                        "ubuntu": "\uf31c",
                        "wsl": "\ue712",
                        "wsl_separator": " on ",
                        "windows": "\ue70f",
                        "postfix": "<#7A7A7A> \ue0b1</>"
                    }
                },
                {
                    "type": "session",
                    "style": "powerline",
                    "powerline_symbol": "\ue0b0",
                    "foreground": "#fafafa",
                    "background": "#2f2f2f",
                    "properties": {
                        "user_info_separator": "<#7A7A7A>\uf1fa</>",
                        "display_default": false,
                        "user_color": "#77f5d6",
                        "host_color": "#2EEFBF",
                        "postfix": "<#7A7A7A> \ue0b1</>"
                    }
                },
                {
                    "type": "root",
                    "style": "powerline",
                    "powerline_symbol": "\ue0b0",
                    "foreground": "#ffff66",
                    "background": "#2f2f2f",
                    "properties": {
                        "root_icon": "\uF0E7",
                        "postfix": "<#7A7A7A> \ue0b1</>"
                    }
                },
                {
                    "type": "path",
                    "style": "powerline",
                    "powerline_symbol": "\ue0b0",
                    "foreground": "#fafafa",
                    "background": "#2f2f2f",
                    "properties": {
                        "style": "agnoster_short",
                        "enable_hyperlink": true,
                        "home_icon": "\uf015 ",
                        "folder_icon": "\ue5fe ",
                        "folder_separator_icon": "\ue0bf "
                    }
                },
                {
                    "type": "git",
                    "style": "powerline",
                    "powerline_symbol": "\ue0b0",
                    "foreground": "#ffeb3b",
                    "background": "#2f2f2f",
                    "properties": {
                        "display_stash_count": true,
                        "display_upstream_icon": true,
                        "display_status": true,
                        "status_colors_enabled": true,
                        "color_background": false,
                        "local_changes_color": "#ffeb3b",
                        "working_color": "#E84855",
                        "staging_color": "#2FDA4E",
                        "ahead_color": "#2EC4B6",
                        "behind_color": "#8A4FFF",
                        "prefix": "<#7A7A7A>\ue0b1 </>"
                    }
                }
            ]
        },
        {
            "type": "prompt",
            "alignment": "right",
            "segments": [
                {
                    "type": "kubectl",
                    "style": "powerline",
                    "powerline_symbol": "\uE0B2",
                    "invert_powerline": true,
                    "foreground": "#3069DE",
                    "background": "#2f2f2f",
                    "properties": {
                        "template": "{{.Context}}{{if .Namespace}} :: {{.Namespace}}{{end}}",
                        "prefix": " \uFD31 ",
                        "postfix": "<#7A7A7A> \ue0b3</>"
                    }
                },
                {
                    "type": "aws",
                    "style": "powerline",
                    "powerline_symbol": "\uE0B2",
                    "invert_powerline": true,
                    "foreground": "#FFA400",
                    "background": "#2f2f2f",
                    "properties": {
                        "template": "{{.Profile}}{{if .Region}}@{{.Region}}{{end}}",
                        "prefix": " \uE7AD ",
                        "postfix": "<#7A7A7A> \ue0b3</>"
                    }
                },
                {
                    "type": "az",
                    "style": "powerline",
                    "powerline_symbol": "\uE0B2",
                    "invert_powerline": true,
                    "foreground": "#9ec3f0",
                    "background": "#2f2f2f",
                    "properties": {
                        "display_id": true,
                        "display_name": true,
                        "info_separator": " @ ",
                        "prefix": " \uFD03 ",
                        "postfix": "<#7A7A7A> \ue0b3</>"
                    }
                },
                {
                    "type": "azfunc",
                    "style": "powerline",
                    "powerline_symbol": "\uE0B2",
                    "invert_powerline": true,
                    "foreground": "#FEAC19",
                    "background": "#2f2f2f",
                    "properties": {
                        "display_version": true,
                        "display_mode": "files",
                        "prefix": " \uf0e7 ",
                        "postfix": "<#7A7A7A> \ue0b3</>"
                    }
                },
                {
                    "type": "dart",
                    "style": "powerline",
                    "powerline_symbol": "\uE0B2",
                    "invert_powerline": true,
                    "foreground": "#06A4CE",
                    "background": "#2f2f2f",
                    "properties": {
                        "prefix": " \uE798 ",
                        "postfix": "<#7A7A7A> \ue0b3</>"
                    }
                },
                {
                    "type": "julia",
                    "style": "powerline",
                    "powerline_symbol": "\uE0B2",
                    "invert_powerline": true,
                    "foreground": "#4063D8",
                    "background": "#2f2f2f",
                    "properties": {
                        "prefix": " \uE624 ",
                        "postfix": "<#7A7A7A> \ue0b3</>"
                    }
                },
                {
                    "type": "rust",
                    "style": "powerline",
                    "powerline_symbol": "\uE0B2",
                    "invert_powerline": true,
                    "foreground": "#B7410E",
                    "background": "#2f2f2f",
                    "properties": {
                        "prefix": " \uE7a8 ",
                        "postfix": "<#7A7A7A> \ue0b3</>"
                    }
                },
                {
                    "type": "java",
                    "style": "powerline",
                    "powerline_symbol": "\uE0B2",
                    "invert_powerline": true,
                    "foreground": "#f89820",
                    "background": "#2f2f2f",
                    "properties": {
                        "prefix": " \uE738 ",
                        "postfix": "<#7A7A7A> \ue0b3</>"
                    }
                },
                {
                    "type": "go",
                    "style": "powerline",
                    "powerline_symbol": "\uE0B2",
                    "invert_powerline": true,
                    "foreground": "#7FD5EA",
                    "background": "#2f2f2f",
                    "properties": {
                        "prefix": " \uFCD1 ",
                        "postfix": "<#7A7A7A> \ue0b3</>"
                    }
                },
                {
                    "type": "ruby",
                    "style": "powerline",
                    "powerline_symbol": "\uE0B2",
                    "invert_powerline": true,
                    "foreground": "#EC0000",
                    "background": "#2f2f2f",
                    "properties": {
                        "prefix": " \uE791 ",
                        "postfix": "<#7A7A7A> \ue0b3</>"
                    }
                },
                {
                    "type": "node",
                    "style": "powerline",
                    "powerline_symbol": "\uE0B2",
                    "invert_powerline": true,
                    "foreground": "#6CA35E",
                    "background": "#2f2f2f",
                    "properties": {
                        "prefix": " \uE718 ",
                        "postfix": "<#7A7A7A> \ue0b3</>"
                    }
                },
                {
                    "type": "python",
                    "style": "powerline",
                    "powerline_symbol": "\uE0B2",
                    "invert_powerline": true,
                    "foreground": "#96E072",
                    "background": "#2f2f2f",
                    "properties": {
                        "prefix": " \uE235 ",
                        "postfix": "<#7A7A7A> \ue0b3</>"
                    }
                },
                {
                    "type": "dotnet",
                    "style": "powerline",
                    "powerline_symbol": "\uE0B2",
                    "invert_powerline": true,
                    "foreground": "#3891A6",
                    "background": "#2f2f2f",
                    "properties": {
                        "prefix": " \uE77F ",
                        "postfix": "<#7A7A7A> \ue0b3</>"
                    }
                },
                {
                    "type": "shell",
                    "style": "powerline",
                    "powerline_symbol": "\uE0B2",
                    "invert_powerline": true,
                    "foreground": "#ffffff",
                    "background": "#2f2f2f",
                    "properties": {
                        "prefix": " \uFCB5 ",
                        "postfix": "<#7A7A7A> \ue0b3</>"
                    }
                },
                // {
                //     "type": "battery",
                //     "style": "powerline",
                //     "powerline_symbol": "\uE0B2",
                //     "invert_powerline": true,
                //     "foreground": "#ffeb3b",
                //     "background": "#2f2f2f",
                //     "properties": {
                //         "battery_icon": "",
                //         "discharging_icon": "\uE231 ",
                //         "charging_icon": "\uE234 ",
                //         "charged_icon": "\uE22F ",
                //         "color_background": false,
                //         "charged_color": "#4caf50",
                //         "charging_color": "#40c4ff",
                //         "discharging_color": "#ff5722",
                //         "postfix": "<#7A7A7A> \ue0b3</>",
                //         "display_charging": true
                //     }
                // },
                {
                    "type": "time",
                    "style": "diamond",
                    "trailing_diamond": "\u2593\ue0c6",
                    "foreground": "#fafafa",
                    "background": "#2f2f2f",
                    "properties": {
                        "time_format": "15:04:05",
                        "postfix": "<#007ACC> \uf64f </>"
                    }
                }
            ]
        },
        {
            "type": "prompt",
            "alignment": "left",
            "segments": [
                {
                    "type": "exit",
                    "style": "plain",
                    "foreground": "#9FD356",
                    "properties": {
                        "display_exit_code": false,
                        "always_numeric": true,
                        "always_enabled": true,
                        "error_color": "#E84855",
                        "color_background": false,
                        "prefix": " \u276F"
                    }
                }
            ]
        }
    ]
}
```

### What OS are you seeing the problem on?

Linux

### Which shell are you using?

bash / zsh

**Comment by JanDeDobbeleer** [comment]
Something's not going to be properly escaped. I'll check what we might have missed.

**Comment by lnu** [comment]
Adding ` "newline": true` on the last prompt fixes the issue

**Comment by daephx** [comment]
That was easier than I thought and perhaps an oversight on my part, thanks @lnu 

I recently updated my prompt referencing slim because it was the one I started with a few versions ago.
And if I recall, I had something similar in the config where `"newline": true`  was it's own entry in the `block: []` list.

I probably just removed it not seeing it present in the newer base theme and missed it being referenced in the last prompt.
Thanks again, that seems to be all I needed!
